apps/interpolate/filter.cpp

```diff
@@ -0,0 +1,40 @@
+#include <cassert>
```

Comment:
A low-priority thought for followup work: it would be nice to have a consistent naming convention for the apps; we have variants of `filter.cpp`, `test.cpp`, `run.cpp`, `process.cpp`, and probably others, with no conventions I can see.

Reply:
At the dawn of history, bilateral_grid used filter, local_laplacian used process, and blur used test. Apps since then have copied one of the above, resulting in the factions seen today. There was never any good reason for these to be different.